src/credential/issuance/04-complete-user-authorization.ts

```diff
@@ -0,0 +1,28 @@
+import type { Out } from "../../utils/misc";
+import type { StartUserAuthorization } from "./03-start-user-authorization";
+
+/**
+ * The interface of the phase to complete User authorization.
+ * It may be implemented as a Credential presentation
+ * or with a strong User identification
+ *
+ * @param requestUri The url to reach to complete the user authorization.
+ * @param cliendId Identifies the current client across all the requests of the issuing flow
+ *
+ * @returns the access code to use to request the credental
+ */
+export type CompleteUserAuthorization = (
+  requestUri: Out<StartUserAuthorization>["requestUri"],
+  clientId: Out<StartUserAuthorization>["clientId"]
+) => Promise<{ code: string }>;
+
+/**
+ * A dummy implementation of CompleteUserAuthorization that uses static values.
+ * Used to replace unimplemented specifications by the Issuer
+ */
+export const completeUserAuthorizationNoOp: CompleteUserAuthorization = async (
+  _,
+  __
+) => {
+  return { code: "static_code" };
+};
```

Comment:
Why is this specification not yet implemented? Could you detail it better? Maybe we also add a TODO with a reference task

Reply:
I agree. I removed this function and implemented _no-op_ in the example app in https://github.com/pagopa/io-react-native-wallet/pull/62/commits/eb3a27676c9120b747d2256766b8479fb2a2c23d